src/web/EmployerFavouritesWeb/Controllers/ApprenticeshipsController.cs

```diff
@@ -42,63 +42,13 @@ public class ApprenticeshipsController : Controller
         [HttpGet("accounts/{employerAccountId:minlength(6)}/apprenticeships")]
         public async Task<IActionResult> Index(string employerAccountId)
         {
-            if (!_paramValidator.IsValidEmployerAccountId(employerAccountId))
-            {
-                _logger.LogDebug($"Invalid parameter: {nameof(employerAccountId)}({employerAccountId})");
-
-                return BadRequest();
-            }
-
-            var response = await _mediator.Send(new ViewEmployerFavouritesQuery
-            {
-                EmployerAccountId = employerAccountId
-            });
-
-            var mapper = new ApprenticeshipFavouriteMapper(_fatConfig);
-
-            var items = response.EmployerFavourites.Select(mapper.Map).ToList();
-
-            items.ForEach(apprenticeship =>
-            {
-                apprenticeship.CreateVacancyUrl = $"{string.Format(_externalLinks.CreateVacancyUrl, employerAccountId)}?ProgrammeId={ apprenticeship.Id }";
-            });
-
-            var model = new ApprenticeshipFavouritesViewModel
-            {
-                EmployerAccountId = employerAccountId,
-                Items = items,
-                HasLegalEntity = response.EmployerAccount.HasLegalEntities
-            };
-            
-            return View(model);
+            return RedirectPermanent(_externalLinks.AccountsHomePage);
```

Comment:
It might be better to use the `AccountsDashboardPage` and format with the `employerAccountId`